llvm/lib/CodeGen/SelectionDAG/SelectionDAGISel.cpp

```diff
@@ -2111,24 +2111,27 @@ bool SelectionDAGISel::CheckOrMask(SDValue LHS, ConstantSDNode *RHS,
 /// by tblgen.  Others should not call it.
 void SelectionDAGISel::SelectInlineAsmMemoryOperands(std::vector<SDValue> &Ops,
                                                      const SDLoc &DL) {
-  std::vector<SDValue> InOps;
-  std::swap(InOps, Ops);
+  // change the vector of SDValue into a list of SDNodeHandle for x86 might call
```

Comment:
This should be a complete sentence starting with a capital letter and ending with a period.